docs/package/gluon-web-logging.rst

```diff
@@ -0,0 +1,5 @@
+gluon-web-logging
+=================
+
+The *gluon-web-logging* package add a new section to advanced settings,
+to allow GUI based configuration of a remote syslog server.
```

Comment:
I'd write "GUI-based" with a dash. Or maybe drop it completely, and just write "allow configuration"?